README.org

```diff
@@ -69,3 +69,16 @@ displays:
 #+begin_quote
 [ Info: Received new data from Julia Symbol Server.
 #+end_quote
+
+*** Do I have to wait for a new release to update the server's dependecies?
+No, you can manually update the self-contained julia environment that eglot-jl
+is configured to create by default. Using the REPL, as an example:
+
+#+begin_src julia
+julia> using Pkg
+julia> cd("/home/username/.emacs.d/elpa/eglot-jl-20211208.359/") # where your version of eglot-jl is
+julia> Pkg.activate(".")
+julia> Pkg.update()
+#+end_src
+
+This will update the dependencies of the server to latest compatible version.
```

Comment:
Could you change this to make clear that the LanguageServer.jl won't be updated if its major version has increased? In that case, the new major version needs to be added to the `compat` section of `Project.toml`, and additional changes may be necessary depending on what the breaking change to LanguageServer.jl was.

Reply:
Of course, that should be clearly stated.